lib/msal-node/src/client/ManagedIdentityApplication.ts

```diff
@@ -183,4 +186,8 @@ export class ManagedIdentityApplication {
             );
         }
     }
+
+    public getManagedIdentitySource(): AzureIdentitySdkManagedIdentitySourceNames {
```

Comment:
Why doesn't this just return a string?

Reply:
It does, essentially. The AzureIdentitySdkManagedIdentitySourceNames is a string enum. This is just extra type checking.